Kanchan-Wakchaure/SER517-Group2-Fall2019
# Added route preview for events

To test:

Create multiple events for the date.
Navigate to preview route or to view events - Click on preview button at the top right corner.
See the marker moving to events created in order.

Scope: Does not simulate route directions, only animates the sequence of events.
